connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceTask.java

```diff
@@ -266,7 +262,29 @@ private Map<TopicPartition, Long> loadOffsets(Set<TopicPartition> topicPartition
     private Long loadOffset(TopicPartition topicPartition) {
         Map<String, Object> wrappedPartition = MirrorUtils.wrapPartition(topicPartition, sourceClusterAlias);
         Map<String, Object> wrappedOffset = context.offsetStorageReader().offset(wrappedPartition);
-        return MirrorUtils.unwrapOffset(wrappedOffset) + 1;
+        return MirrorUtils.unwrapOffset(wrappedOffset);
+    }
+
+    // visible for testing
+    void initializeConsumer(Set<TopicPartition> taskTopicPartitions) {
+        Map<TopicPartition, Long> topicPartitionOffsets = loadOffsets(taskTopicPartitions);
+        consumer.assign(topicPartitionOffsets.keySet());
+        log.info("Starting with {} previously uncommitted partitions.", topicPartitionOffsets.values().stream()
```

Comment:
IMO the existing log is fine--can you elaborate on why you believe this change is necessary? Logging at INFO level seems okay since this should happen once per task lifecycle, and "Starting with" also seems warranted given that this should only occur during `MirrorSourceTask::start`.

Reply:
I just thought that we already have an info level message at the end of `MirrorSourceTask::start` and this message was not that useful, but I don't have a strong opinion, so I'm fine with leaving it as it is.